services/static-webserver/client/source/class/osparc/po/PreRegistration.js

```diff
@@ -76,11 +78,27 @@ qx.Class.define("osparc.po.PreRegistration", {
         }
         if (form.validate()) {
           submitBtn.setFetching(true);
+
+          const flashErrorMsg = this.tr("Pre-Registration Failed. See details below");
```

Comment:
if I'm not mistaken this should either be Preregistration or Pre-registration but your call